docs: update CLI Usage page in the Sphinx documentation (#269)
oracle-samples/macaron: docs/source/pages/installation.rst

.. code-block:: shell

>   ./run_macaron.sh macaron --help

Reviewer: No need to add `macaron` (similar in other places):
```suggestion
  ./run_macaron.sh --help
```

Author: Addressed in https://github.com/oracle-samples/macaron/pull/269/commits/fc12961486572c6e91d62fa014813a49bf96df07

.. note:: In the first execution, this script will download the Macaron Docker image from ``ghcr.io/oracle-samples/macaron`` which can take some time. However, the next time you run it, the docker image available on your local host will be used.

> .. note:: By default, ``latest`` is used as the tag for the downloaded image. You could specify the tag you want to run by assigning the environment variable ``MACARON_IMAGE_TAG``. For example to run Macaron v0.1: ``MACARON_IMAGE_TAG=0.1 && ./run_macaron.sh macaron --help``

Reviewer: ```suggestion
.. note:: By default, ``latest`` is used as the tag for the downloaded image. You can choose a specific tag by assigning the environment variable ``MACARON_IMAGE_TAG``. For example to run Macaron v0.1.0 run: ``MACARON_IMAGE_TAG=v0.1.0 && ./run_macaron.sh --help``

Not that the image tag starts with `v`.
```

Author: By ``Not that the image tag starts with `v`.`` do you mean ``Note that the image tag starts with `-v`.`` Do you think we should use a different `.. note::` derivative for it or we could put it directly after the previous `.. note::`?

Reviewer: > By `` Not that the image tag starts with `v`. ``

Yes, sorry :sweat_smile: That was just for your information. No need to add it as a note.

Author: Oh I see. No worries :grin: Thank you.

Author: Addressed in https://github.com/oracle-samples/macaron/pull/269/commits/602d68376381eb591484d8df2ecaee0e26dd0e7e
